didx-xyz/yoma-mobile
# YOMA-455 user skills cv display all skills in view

## Scope [YOMA-455](https://yomaworld.atlassian.net/browse/YOMA-455)
 - get skills into MySkills view
 
![](https://media.giphy.com/media/XfyyRZhUNwcb31WUH8/giphy.gif)
## Work Done
- adjusted current MySkills to work with a form view and a list view
- moved MySkills list view to use the View component, CvViewSkills, etc

Note - it's partially applied to the current designs.

## Steps to Test
- run tests
- check that everything is viewing correctly
![Screenshot 2021-09-12 at 22 16 40](https://user-images.githubusercontent.com/1424561/133001571-8989f8a1-fcc8-4498-b931-eadf65d06c3b.png)

    
## Screenshots
